deleteDiscordMessages.js

```diff
@@ -224,18 +230,18 @@
             log.verb(`Estimated time remaining: ${etr}`)
             
             
-            if (myMessages.length > 0) {
+            if (messagesToDelete.length > 0) {
```

Comment:
I think this was supposed to be `deletableMessages`, rather than `myMessages`. Otherwise we might try to delete messages when none can be deleted.